pkg/operator/ceph/config/keyring/cephx.go

```diff
@@ -26,7 +26,13 @@ import (
 	"github.com/rook/rook/pkg/operator/ceph/version"
 )
 
-var CephAuthRotateSupportedVersion = version.CephVersion{Major: 20, Minor: 2, Extra: 0}
+// CephAuthRotateSupportedVersion identifies the first ceph release in which the `ceph auth rotate`
+// command is present, thus allowing CephX key rotation.
+var CephAuthRotateSupportedVersion = version.CephVersion{Major: 19, Minor: 2, Extra: 3}
+
+// CephAuthMonRotateSupportedVersion identifies the first ceph release in which `ceph auth rotate`
+// can be used to rotate the monitor CephX key.
+var CephAuthMonRotateSupportedVersion = version.CephVersion{Major: 20, Minor: 3, Extra: 0} // v20.3.0 is unreleased tentacle vers
```

Comment:
I'd expect 20.2.0 to be the first official tentacle release.
```suggestion
var CephAuthMonRotateSupportedVersion = version.CephVersion{Major: 20, Minor: 2, Extra: 0} // v20.2.0 is unreleased tentacle vers
```

Reply:
20.**3**.0 is the unreleased tentacle version. Keeping it at 20.3.0 means that we don't have to worry about tentacle being released without this support

Reply:
It's 20.3.0. This will work with the prerelease images the team has been working with, and it will ensure that other images (even a tentacle main release) won't work until we are able to test it and enable it.

```
[root@dd37b2675a13 /]# ceph --version
ceph version 20.3.0-2332-ga3cf73b8 (a3cf73b810d334fbf1819adc3be592d1ea49aa93) tentacle (dev)
```